icons/bot.json

```diff
@@ -1,7 +1,6 @@
 {
   "$schema": "../icon.schema.json",
   "contributors": [
-    "mittalyashu",
```

Comment:
Why remove them?

Reply:
Because their entire contribution to this icon has essentially been removed?

Reply:
Okay 👍